open-metadata-implementation/user-interfaces/presentation-server/ssl/keys/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
this is reintroducing a different key - we've just fixed this for 2.3 and are using the same key as the rest of the UI - for now. recommend this change is removed

Reply:
or at least corrected to use same file
it's also possible to copy the file by using the maven resources plugin in the pom - but that can be an optimization for later when we finess security

Reply:
updated key to use one in master branch